### PR TITLE
[GSoC Proposal] IJCLab / AstroLab Software / Big data & Graph database

### PR DESCRIPTION
**Pull request by JulienPeloton**
Dear HSF GSoC 2020 admin team,

You will find a new proposal from the organisation **IJCLab**, and the project **AstroLab** (participating project at GSoC 2018, 2019).

Note that IJCLab was formerly known as LAL (participating organisation at GSoC 2017, 2018, 2019) - but since 1st January 2020 the LAL has joined four laboratories located on the Orsay campus of the Paris-Saclay University to become the _Laboratoire de Physique des 2 infinis Irène Joliot-Curie_ (IJCLab). I then pushed a new organization description, and a logo (the official logo is not known, so it is just the name of the lab).

@hrivnac, @ChrisArnault

### CODE REVIEW

**Comment by xvallspl** [comment]
@JulienPeloton I've just seen that you didn't commit the new logo. Can you solve this, please?

**Comment by JulienPeloton** [comment]
Thanks @xvallspl, the logo has been committed!

File: _gsocprojects/2020/project_AstroLab.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+project: AstroLab
+layout: default
+logo: Astrolab-logo.png
+description: |
+  [AstroLab Software](https://astrolabsoftware.github.io/) is a project from [IJCLab](https://www.lal.in2p3.fr/en/) aiming at providing advanced software tools to overcome modern science challenges faced by research groups, and allow research communities to more fully exploit the big data ecosystem tools.
+summary: |
```
Comment (agheata) [inline _gsocprojects/2020/project_AstroLab.md]: when (summary == description) no need to add it.

Comment (JulienPeloton) [reply _gsocprojects/2020/project_AstroLab.md]: Thanks for the tip @agheata !

**Pull request merged**